src/OpenTelemetry.Instrumentation.Quartz/QuartzInstrumentationOptions.cs

```diff
@@ -43,7 +43,7 @@ public class QuartzInstrumentationOptions
     /// <para>object: the raw object from which additional information can be extracted to enrich the activity.
     /// The type of this object depends on the event, which is given by the above parameter.</para>
     /// </remarks>
-    public Action<Activity, string, object> Enrich { get; set; }
+    public Action<Activity, string, object?>? Enrich { get; set; }
```

Comment:
I think the better thing to do would be to not mark `object` nullable and update the code to call `Enrich` only when the `object` passed to it is not null.

Reply:
Please check 8b32c4b3837e37eeb2bfef7edc5e16d414a58e57

Reply:
I think the update is fair.